sources/packages/backend/workflow/test/2021-2022/parttime-assessment/awards-amounts/parttime-assessment-2021-2022-awards-amount-CSPT.e2e-spec.ts

```diff
@@ -0,0 +1,128 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+import { YesNoOptions } from "@sims/test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-CSPT.`, () => {
+  it("Should determine awardEligibilityCSPT when calculatedDataTotalFamilyIncome <= limitAwardCSPTIncomeCap", async () => {
```

Comment:
Are these tests intended to validate the award amounts or the eligibility?
I am struggling to understand the E2E purpose while reading the description and figuring out what the tests are validating.

Reply:
Updated it, thanks for noting.